externs/shaka/player.js

```diff
@@ -813,6 +814,11 @@ shaka.extern.HlsManifestConfiguration;
  *   configured or set as 0.
  *   For HLS, the default value is 3 segments duration if not configured or
  *   set as 0.
+ * @property {boolean} textOffsetMode
```

Comment:
Thanks, I thought it might be inappropriate to be explicit about adding a VTT-only option and have code moving it across player/media source engine/text engine/etc. 

What if more text settings have to be introduced in the future? Maybe I could introduce some `textEngine.segmentRelativeVttTiming` so more settings could be added and we would not be adding code just to pass one VTT-only flag?

Reply:
Done @joeyparrish